src/LogikUI/Interop/Logic.cs

```diff
@@ -9,14 +9,36 @@ namespace LogikUI.Interop
 {
     static class Logic
     {
+#if WINDOWS
```

Comment:
In modern c# there exists a better solution for this where you don't need static ifs. 
I can implement this after this pr is merged though,